backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/docvalues/impl/FieldData.java

```diff
@@ -0,0 +1,339 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.lowlevel.docvalues.impl;
+
+import java.io.IOException;
+import org.apache.lucene.index.DocValues;
+import org.apache.lucene.index.NumericDocValues;
+import org.apache.lucene.index.SortedNumericDocValues;
+
+/**
+ * Utility methods, similar to Lucene's {@link DocValues}.
```

Comment:
Same here, please add a few lines to give credit.

Reply:
Corrected